src/math/Kconfig

```diff
@@ -22,15 +22,35 @@ config POWER_FIXED
 
 
 config BINARY_LOGARITHM_FIXED
-       bool "Binary Logarithm function"
-       default n
-       help
+	bool "Binary Logarithm function"
+	default n
+	help
 	  This option builds Binary Logarithm function (log2n) is the logarithm to
 	  the base 2. The binary logarithm of x is the power to which the number 2
 	  must be raised to obtain the value x. Base 2 logarithm function is implemented
 	  with a short lookup table. (log2n) operates for a range of 32 bit width size
 	  i.e. 1 to 4294967295.
 
+config NATURAL_LOGARITHM_FIXED
+	bool "Natural Logarithm function"
+	default n
+	help
+	  This option builds Natural Logarithm function (loge(N)) is the logarithm to
+	  the base e. The natural logarithm of x is the power to which the number e
+	  must be raised to obtain the value x. Base e logarithm function is implemented
+	  with a short lookup table. (loge(N)) operates for a range of 32 bit width size
+	  i.e. 1 to 4294967295.
+
+config COMMON_LOGARITHM_FIXED
+       bool "Common Logarithm function"
+       default n
+       help
+	  This option builds common Logarithm function (log10(N)) is the logarithm to
+	  the base 10. The common logarithm of x is the power to which the number 10
+	  must be raised to obtain the value x. Base 10 logarithm function is implemented
+	  with a short lookup table. (log10(N)) operates for a range of 32 bit width size
```

Comment:
implementation details should be comments inside the c file, not part of the config

Reply:
do I change anything here ?

Reply:
See below.

Reply:
s/below/above